baseline-refaster-rules/src/main/java/com/palantir/baseline/refaster/AssertjMapDoesNotContainKey.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.baseline.refaster;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.google.errorprone.refaster.ImportPolicy;
+import com.google.errorprone.refaster.annotation.AfterTemplate;
+import com.google.errorprone.refaster.annotation.BeforeTemplate;
+import com.google.errorprone.refaster.annotation.UseImportPolicy;
+import java.util.Map;
+
+public final class AssertjMapDoesNotContainKey<K, V> {
+
+    @BeforeTemplate
+    void before1(Map<K, V> things, K key) {
+        assertThat(things.containsKey(key)).isFalse();
+    }
+
+    @BeforeTemplate
+    void before2(Map<K, V> things, K key) {
+        assertThat(!things.containsKey(key)).isTrue();
+    }
+
+    @BeforeTemplate
+    void before3(Map<K, V> things, K key) {
+        assertThat(things.get(key)).isNull();
+    }
+
+    @BeforeTemplate
+    @SuppressWarnings("RedundantCollectionOperation") // It's what we're fixing
+    void before4(Map<K, V> things, K key) {
+        assertThat(things.keySet().contains(key)).isFalse();
+    }
+
+    @BeforeTemplate
+    @SuppressWarnings("RedundantCollectionOperation") // It's what we're fixing
+    void before5(Map<K, V> things, K key) {
+        assertThat(!things.keySet().contains(key)).isTrue();
```

Comment:
If we had a standalone refaster that changed `assertThat(!arbitaryBoolean).isFalse()` to `assertThat(arbitaryBoolean).isTrue()`, we wouldn't need this case right?

Reply:
Yep, I can add that

Reply:
If we're doing silly ones, might want `assertThat(foo == null).isTrue()` -> `.isNull()` too?

Reply:
Removed the negation checks from this class and added the general rule.